amundsen_application/static/css/_list-group.scss

```diff
@@ -7,7 +7,7 @@
     border-right: none;
     padding: 0;
 
-    &:hover {
+    &.clickable:hover {
```

Comment:
Just a thought, but since it  like most of our list groups will actually be clickable, what do you think of making it such that items have to opt-out of this style as opposed to opt-in? Fine with either way for now.

Reply:
I thought about that too-- even though most list-group-items in Amundsen happen to be clickable at the moment, list-group-items in general tend not to be clickable by default. 

I think it's more intuitive to add behavior with special classes rather than take them away with negative classes. For example bootstrap has the optional `list-group-item-action` class that you can add to normal `list-group-item`... Now that I think about it, we _could_ use that class but then we'd have to overwrite all of the default styles as well.